catwalk/storage.py

```diff
@@ -129,7 +165,7 @@ def uuid(self):
         return self.metadata['metta-uuid']
 
     def columns(self, include_label=False):
-        columns = self.matrix.columns.tolist()
+        columns = self._matrix.columns.tolist()
```

Comment:
I think you just want to remove this old version of the method? Unless the diff is confusing me, you have the correct new implementation in the same class up above (line 142)

Reply:
woops! so I made another small changes in test_storage for more attribute testing.